upgrading/automated_upgrades.adoc

```diff
@@ -170,18 +170,28 @@ with {product-title} 3.9:
 ----
 
 .. In previous versions of {product-title}, master hosts were marked unschedulable
-by default by the installer, meaning that new pods could be placed on the hosts.
-Starting with {product-title} 3.9, however, masters must be marked schedulable;
-to do so, set the `openshift_schedulable=true` variable for each master host
-listed in the `[nodes]` group of your inventory file:
+by default by the installer, meaning that no pods could be placed on the hosts.
+Starting with {product-title} 3.9, however, masters must be marked schedulable
+and this is done automatically during upgrade. To do so manually, set the
+`openshift_schedulable=true` variable for each master host listed in the
+`[nodes]` group of your inventory file:
 +
 ----
 [nodes]
 master.example.com openshift_schedulable=true
 ----
+
+.. As of {product-title} 3.9, the web console is removed from the master services
+and deployed as a pod on the masters. Modify the *_master-config.yaml_* to add
+`defaultNodeSelector`:
 +
-This change is mainly so that the web console, which used to run as part of the
-master itself, can instead be run as a pod deployed to the master.
+----
+# cat /etc/origin/master/master-config.yaml|grep "defaultNodeSelector"
+  defaultNodeSelector: node-role.kubernetes.io/compute=true
+----
+
+.. Add the `node-role.kubernetes.io/compute=true` label for non-infra and
+non-masters nodes.
```

Comment:
@ahardin-rh Hi, Ashley. 

The steps above should be part of manual upgrade, NOT automated upgrade. Also the command and statement is a draft only as it's really complicated and challenging that QE can't offer the details.

Reply:
Yes, @ganhuang is correct, this is handled automatically via the playbooks.